biolink-model.yaml

```diff
@@ -2219,6 +2282,97 @@ slots:
     mappings:
       - rdf:type
 
+  ## Additional Properties for Information Content Entity and Publication Nodes
+
+  published in:
+    is_a: node property
+    description: >-
+      CURIE identifier of a broader publication context within which the publication may be placed,
```

Comment:
Consider whether 'publication' is overloaded here, when it comes to things like journals and articles they contain.  Is the journal the publication, or each article a publication, or both?  From the use of this term here, and the definition of the Publication class below, it sounds like it is meant broadly to cover both.

Reply:
Good morning @mbrush (cc: @cmungall) **Short answer:** yes, it is overloaded. 

In reading some public models for "Publications", this seems an acceptable idea in some quarters.

That said, maybe it is overmodelling in Biolink for our purposes in Translator. I sensed that was the opinion expressed by others (I think including Chris M. and Deepak).  I am ok with that.

**Option 1:** dispense completely with instantiated concept nodes for `Journal` and `Book` categories, and rather just use the ISSN and ISBN codes to identify them using the `published in` property (plus perhaps, the `iso abbreviation` property filled in for every journal articles).  Then Leave access to other details about the journal or book to the consumer of the knowledge, outside of Translator knowledge graphs.

**Option 2:** define standalone (`is_a: named thing`) `Journal` and `Book` Biolink categories with minimalist properties. Once again, just use the ISSN and ISBN codes to point to them using the `published in` property, but some basic entity details can be tracked including perhaps, the `iso abbreviation` property filled in for each journal (maybe don't embed this field in the journal article itself).

**Option 3:**  would be to insert a more lightweight parent class for a collection of publication subtypes  which might have common subclasses like `Serial` (or `Journal`) and `Book` as siblings to `Publication` or perhaps, give `Journal Article` and `Book Chapter` category classes first class modelling status, inheriting from the parent as siblings to `Serial` and `Journal`.  

I'm not sure what to call the parent class, if not `Publication`... please express your opinion about this, if this is an acceptable design option.  

Once again, the ISSN and ISBN codes could still identify concept nodes for a given Journal and Article, for dereferencing in the `published in` property of the child publications.

**Option 4:** some other model design?

If any other specialized publication types are encountered in Translator, we might give them a dedicated category or simply figure out how to model them in a 'catch all' publication category class, suitably tagged with publication type and annotated with generic `has attribute` metadata to capture additional details required to capture the necessary semantics of such oddball publication subtypes.

Reply:
In the absence of any clear use cases for needing to describe attributes of things like Journals/Serials/Periodicals in our data, I lean toward the simplest option 1.  If such use cases arise, we could add some class to represent a parent class for journals/periodicals so they can be more richly described.  Or, we can explicitly broaden our Publication class to include discrete published works and the larger periodicals/journals of which they are a part, and add new fields to capture attributes of the larger periodicals/journals (e.g. ISBN code, etc.).

Reply:
I actually opted for Option 3, creating various `Publication `subtypes, all inheriting common slots from Publication by inheritance (remember that all such slots are "optional" in any case, unless `slot_usage` specifies otherwise) and pushing down more   I've created `BookChapter` and `Article` (i.e. journal article) to capture the two important publication subtypes. I kept pages and descriptive metadata (keywords, abstract, etc) in the parent `Publication` field, even though the latter may be rarely expressed annotation for some publication subtypes.  The two primary parent contexts - Book and Serial - don't need to be used in a given subgraph, but are there if use cases need them for some reason.